submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/mapping/to/ToExtFileListMapper.kt

```diff
@@ -17,25 +17,27 @@ class ToExtFileListMapper(
      * Maps a DocFileList to corresponding Ext type. Note that empty list is used if includeFileListFiles is false as
      * files list files are not loaded as part of the submission.
      */
+    @Suppress("LongParameterList")
     fun toExtFileList(
         fileList: DocFileList,
         subAccNo: String,
         subVersion: Int,
+        released: Boolean,
```

Comment:
I understand these two fields are required here because we currently don't have that information in the database for old files so I think we should at least add a TODO here to remind us to remove them and add them to the doc files themselves once we do the submission files decoupling.